ocs_ci/deployment/vmware.py

```diff
@@ -1120,9 +1524,22 @@ def clone_openshift_installer():
     ocp_version = get_ocp_version()
     # supporting folder structure from ocp4.5
     if Version.coerce(ocp_version) >= Version.coerce("4.5"):
-        clone_repo(
-            constants.VSPHERE_INSTALLER_REPO, upi_repo_path, f"release-{ocp_version}"
-        )
+        if config.ENV_DATA["sno"]:
+            constants.VSPHERE_INSTALLER_REPO = (
+                "https://github.com/shyRozen/installer.git"
+            )
```

Comment:
please open an issue for later modification of this, so we will not use a private repo

Reply:
https://github.com/red-hat-storage/ocs-ci/issues/6040